python/demo/demo_cahn-hilliard.py

```diff
@@ -301,7 +301,7 @@
 # Prepare viewer for plotting the solution during the computation
 if have_pyvista:
     # Create a VTK 'mesh' with 'nodes' at the function dofs
-    topology, cell_types, x = plot.create_vtk_mesh(V0)
+    topology, cell_types, x = plot.create_vtk_mesh(V0, msh.topology.dim)
```

Comment:
Can we make make the second argument to `create_vtk_mesh` have a default  value (`msh.topology.dim`)?